helmctlr/helm_test.go

```diff
@@ -134,9 +155,13 @@ func TestResourceAddedHappyPath(t *testing.T) {
 		create:   1,
 		releases: 1,
 	}
-	assertCounters(t, ct, func() {
-		testController.ResourceAdded(testResource)
-	})
+	tsExpected := make(map[string]func(float64, float64) bool)
```

Comment:
Can we not make a struct that has all of these and defaults them to lessThan? Then we only have to specify the ones that are greaterThan and it ensures all of them are always checked instead of only the ones specified. Similar to what we did for the other metrics. Plus this way if we add a new timestamp we only have to list it in one place to get it everywhere.